src/Features/PlayerTrace.cpp

```diff
@@ -147,12 +152,14 @@ ON_EVENT(PRE_TICK) {
 }
 
 ON_EVENT(SESSION_START) {
-    playerTrace->Clear();
+    if (sar_player_trace_autoclear.GetBool())
+        playerTrace->Clear();
 }
 
 HUD_ELEMENT2(player_trace_draw_speed, "1", "Toggles drawing the speed deltas on the player traces\n", HudType_InGame) {
 
     if (!sar_player_trace_draw.GetBool()) return;
+    if (!sar_player_trace_draw_speed_deltas.GetBool()) return;
```

Comment:
If you have your own toggle for this, you should use HUD_ELEMENT2_NO_DISABLE, as else you'll have a `sar_hud_player_trace_draw_speed` toggle too (which ofc makes no sense lol)